Test1/src/Problem3.kt

```diff
@@ -0,0 +1,56 @@
+// Am incercat sa rezolv in mai multe metode problema deorece nu eram sigur daca prima solutie era cea ideala
+// Rezolvarea ideala cu Stringuri
+fun main() {
+    var a = intArrayOf(1204, 66, 7423, 1, 4, 850, 4, 55, 10003, 55, 78364, 93, 403, 1789, 1, 3850, 778, 850)
+
+    var distincte = ""
+    var duplicate = ""
+
+    for(i in a.indices) {
+        if(i == 0) {
+            distincte += (a[i].toString() + " ")
+        } else if(distincte.contains(" " + a[i])) {
```

Comment:
`contains` most likely uses a loop in it's implementation, so you are still using a loop inside a loop, there is a clear way to solve this problem.